### DISCUSSION TITLE
How to customize commands?

### DISCUSSION THREAD

**Discussion by FZliweiliang** [dlemstra/magick-wasm#84] [Q&A]
Some unsupported features hope to allow users to customize commands

**Comment by dlemstra** [comment]
I am sorry but I don't understand what you are trying to ask me.

**Reply by FZliweiliang** [reply]
I'm sorry if I wasn't clear enough, as in my demo below I wanted to use the command call directly
`ImageMagick(input,{
  commands:[`+profile "*" -strip -quality ${config.quality*100}%`]
})`

**Reply by dlemstra** [reply]
You cannot do that with this library. You will need to use the api instead.
